style-guide/styles/wds-components/_global-navigation.scss

```diff
@@ -1,556 +1,513 @@
+@import 'global-navigation-search';
+
 .wds-global-navigation {
-	$border-top-width: 6px;
-	$logo-width: 230px;
-	$nav-height: $wds-global-navigation-height - $border-top-width;
-	$nav-background-color: $wds-color-white;
 	$search-input-padding: 3px 4px;
 	$notification-counter-size: 18px;
+	$partner-slot-width: 80px;
+	$scroll-animation-duration: .4s;
 
-	$search-transition-duration: 200ms;
-	$search-buttons-transition-duration: 50ms;
-	$search-buttons-transition-delay: $search-transition-duration - $search-buttons-transition-duration;
+	background-color: $wds-fandom-color-navy;
+	box-sizing: border-box;
+	font-family: Rubik,"Helvetica Neue",Helvetica,Arial,sans-serif;
+	left: 0;
+	position: fixed;
+	top: 0;
+	width: 100%;
+	z-index: 3; // 3 to cover the community bar dropdown while scrolling
 
-	// this should be moved to _colors in the future as that color is from the new palette of Fandom colors
-	$new-fandom-logo-color: #00d6d6;
+	@media #{$wds-breakpoint-small-down} {
+		padding: 0 3px 0 12px;
+	}
 
-	@mixin visible-search-input {
-		&__search-input {
-			// transitioned property: padding;
-			padding: $search-input-padding;
-			position: relative;
-			top: auto;
+	&__content-container {
+		margin: 0 auto;
+		display: flex;
+
+		@media #{$wds-breakpoint-medium-up} {
+			width: calc(100vw - 24px)
 		}
-	}
 
-	@mixin search-label-icon-default-hover-color {
-		&__search-label-icon:hover {
-			color: $wds-color-dark-blue-gray;
+		@media #{$wds-breakpoint-l-up} {
```

Comment:
Let's use new breakpoints instead of the old ones, `$wds-breakpoint-l-up` -> `$wds-breakpoint-large-up`, same for all other breakpoints.

Reply:
See also the list here https://fandomdesignsystem.com/#/base-styles/wds-breakpoints

Reply:
Talked off-line 👎